### ISSUE TITLE
Remove not working in Overworld

### ISSUE THREAD

**Issue by davqvist** [McJty/InControl#59]
Hi. 
I used InControl first for all the custom dimensions that I added and it worked beautifully fine. Now I want to remove most mobs from the overworld and none of the removes seems to work. Here is my potentialspawn.json: https://pastebin.com/wPSDVh3D (see lines starting from 32 for the dim 0 removals).
Here is my spawn.json FWIW: https://pastebin.com/1QGxw2Zy

Let me know if you need more information.

Forge 2654
In Control 3.8.0

**Comment by Pyromanc3r** [comment]
You do not need to create a rule "remove" in the file "potentialspawn.json" to delete the mobs.
Just specify the dimension, biome (if necessary) and the mobs in the file "spawn.json".
{
	"dimension": 0,
	"biome": "Plains",
	"mob": [
	"minecraft:bat",
	"minecraft:chicken",
	"minecraft:cow",
	"minecraft:donkey",
	"minecraft:horse",
	"minecraft:llama",
	"minecraft:mule",
	"minecraft:mooshroom",
	"minecraft:ocelot",
	"minecraft:parrot",
	"minecraft:pig",
	"minecraft:rabbit",
	"minecraft:sheep",
	"minecraft:squid",
	"minecraft:villager",
	"minecraft:wolf"
	],
	"result": "deny"
	}
Biome names and entities can be taken by addon BiomeTweaker

**Comment by davqvist** [comment]
Nope, both ways are not working for me. I would be also surprised that deny in spawn would work while remove in potentialspawn doesn't work as the latter works for me fine in all other dimensions than the overworld.

I am just lost on how to track the cause down. The only difference in the overworld is that I have Lost Cities world gen (with Spawners turned off though). Would that be a reason?

**Comment by McJty** [comment]
Most likely explanation is that some other mod is messing with overworld spawns. Might be hard to track down though
